app/javascript/dashboard/components-next/HelpCenter/Pages/ArticleEditorPage/ArticleEditorControls.vue

```diff
@@ -36,13 +38,15 @@ const currentUser = computed(() =>
   agents.value.find(agent => agent.id === currentUserId.value)
 );
 
+const categorySlugFromRoute = computed(() => route.params.categorySlug);
+
 const author = computed(() => {
   if (isNewArticle.value) {
     return selectedAuthorId.value
       ? agents.value.find(agent => agent.id === selectedAuthorId.value)
       : currentUser.value;
   }
-  return props.article?.author || currentUser.value;
+  return props.article?.author || null;
```

Comment:
@pranavrajs Handle the case where the original author is removed from the account. (Previously, it displayed the current user automatically without selection. Now, we show an icon on the thumbnail with a tooltip text 'Author is not available.' This only applies to existing articles.)

**Card**
<img width="358" alt="image" src="https://github.com/user-attachments/assets/29120c33-c69b-4144-b647-0453d3990f9d">


**Editor**
<img width="459" alt="image" src="https://github.com/user-attachments/assets/5e14fb72-a6d1-49de-903f-dafd28f08abd">